child-process.ts

```diff
@@ -49,6 +49,11 @@ export class ChildProcess implements IChildProcess {
 		return child_process.spawn(command, args, options);
 	}
 
+	public fork(modulePath: string, args?: string[], options?: any): child_process.ChildProcess {
+		this.$logger.debug("fork: %s %s", modulePath, args.join(" "));
```

Comment:
This log does not add double quotes around arguments with spaces, which makes debugging very hard. We already have this issue in other child process functions and had to use other methods to get the correct command line.